locales/messages.json

```diff
@@ -47,6 +47,12 @@
   "_ChecksLineInfo.comment": "{Locked}",
   "ChecksUnreachableCode": "unreachable code was reached",
   "ChecksUpdateVcpkg": "updating vcpkg by rerunning bootstrap-vcpkg may resolve this failure.",
+  "CiBaselineRegression": "REGRESSION: {spec} failed with {build_result}. If expected, add {spec}=fail to {path}.",
+  "_CiBaselineRegression.comment": "example of {spec} is 'zlib:x64-windows'.\nexample of {build_result} is 'One of the BuildResultXxx messages (such as BuildResultSucceeded/SUCCEEDED)'.\nexample of {path} is '/foo/bar'.\n",
```

Comment:
this should be `{Locked}`.

Reply:
I wasn't sure if it should be. The "spec=fail" part needs to be but there's no reason the rest of the message couldn't be localized.

Reply:
mmh, okay